model-optimizer/extensions/front/GeLUMerger_Tanh.py

```diff
@@ -74,6 +74,6 @@ def replace_sub_graph(self, graph: Graph, match: dict):
             # check that the values match the approximation
             if fabs(mul0_param - sqrt2pi) < 1e-06 and fabs(mul_param - 0.044715) < 1e-06 and mul1_param == 0.5:
                 log.debug('Confirmed TanH-based GELU pattern after {} with name {}'.format(inp.op, inp.name))
-                gelu = GeLUOP(graph, dict(name=inp.name + '/GELU_')).create_node()
+                gelu = GeLUOP(graph, dict(name=inp.name + '/GELU_', approximation='tanh')).create_node()
```

Comment:
Please, update copyright year in this file.

Reply:
Fixed.